CHANGELOG.md

```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
+
+- [RJMR-267](https://makeitapp.atlassian.net/browse/RJMR-267): added makefile and github actions. Updated dependencies.
```

Comment:
```suggestion
- added makefile and github actions. Updated dependencies.
```

no need for internal tracking logs; also I'd add the releaser action and deprecate the changelog (like we did [here](https://github.com/rond-authz/rond/blob/main/.github/release.yml))

Reply:
Removed entry from changelog and added deprecation warning in it. As suggested, releaser action has been added